### PR TITLE
Fix loader-generator and plugin-generator tests

### PR DESCRIPTION
**Pull request by Loonride**
**What kind of change does this PR introduce?**
test fix

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
N/A

**Summary**
There were some problems with the `loader-generator` and `plugin-generator` test files which I fixed.

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by Loonride** [comment]
> We got rid of lock files since we use yarn workspaces now, please use yarn and remove the lock files.

fixed

**Comment by Loonride** [comment]
> It looks like it's not fixed yet.
> 
> Look at the logs in the CI. I believe the assertion library doesn't work well with jest OR the assertions are not correct.
> 
> ```
> npm ERR! code ENOENT
> npm ERR! syscall open
> npm ERR! path /tmp/c12aae677bfdb92438557a5218666476c5e2f591/my-loader/package.json
> npm ERR! errno -2
> npm ERR! enoent ENOENT: no such file or directory, open '/tmp/c12aae677bfdb92438557a5218666476c5e2f591/my-loader/package.json'
> npm ERR! enoent This is related to npm not being able to find a file.
> npm ERR! enoent 
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/runner/.npm/_logs/2020-02-22T08_18_01_051Z-debug.log
> ```

It turns out this had nothing to do with the test itself, the generators needed to be writing a package.json but were not.

**Comment by anshumanv** [comment]
> It turns out this had nothing to do with the test itself, the generators needed to be writing a package.json but were not.

Exactly, and the weird bit it that the tests are passing, it should fail in case it fails to write the files.

**Comment by ematipico** [comment]
So we've found a bug! Nice!! 🙏

File: packages/generators/src/plugin-generator.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ const PluginGenerator = addonGenerator(
 			validate: (str: string): boolean => str.length > 0
 		}
 	],
-	path.resolve(__dirname, "..", "generate-plugin"),
+	path.resolve(__dirname, "../../generate-plugin/templates"),
```
Comment (ematipico) [inline packages/generators/src/plugin-generator.ts]: Thank you for fixing this! That was my fault

**Pull request merged**